packages/client/src/pages/Profile/index.tsx

```diff
@@ -88,6 +93,15 @@ function ProfilePage() {
                       <Box key={key} className={styles.fieldItem}>
                         <Typography>{fields[key]}</Typography>
                         <TextField
+                          error={
+                            !valid[key].valid.isValid && valid[key].blur.isDirty
+                          }
+                          helperText={
```

Comment:
and here you can use undefined instead of ' '

Reply:
> and here you can use undefined instead of ' '

строчка с пробелом " " ипользуется для резервации места под helperText в MUI. Если его заменить на undefined, оно исчезнет